documentation/asciidoc/topics/con_hotrod_clients.adoc

```diff
@@ -6,16 +6,25 @@ Hot Rod is a binary TCP protocol that {brandname} provides for high-performance
 
 .Client intelligence
 
-The Hot Rod protocol includes a mechanism that provides clients with an up to date view of the cache topology.
+The Hot Rod protocol includes a mechanism that provides clients with an up-to-date view of the cache topology.
 Client intelligence improves performance by reducing the number of network hops for read and write operations.
 
 Clients running in the same {k8s} cluster can access internal IP addresses for {brandname} pods so you can use any client intelligence.
-The default intelligence, `HASH_DISTRIBUTION_AWARE`, is recommended because it allows clients to route requests to primary owners, which improves performance.
 
 [NOTE]
 ====
 {k8s} cluster administrators can define network policies that restrict traffic to {brandname}.
 In some cases network isolation policies can require you to use `BASIC` intelligence even when clients are running in the same {k8s} cluster but a different namespace.
 ====
 
-Hot Rod clients must use `BASIC` intelligence when connecting to {brandname} through a `LoadBalancer`, `NodePort`, or {openshiftshort} `Route`.
+`HASH_DISTRIBUTION_AWARE`:: The default intelligence mechanism. Best mechanism for allowing clients to route requests to primary owners, which improves a client's performance. 
+
+[NOTE]
+====
+Hot Rod clients must use `BASIC` intelligence for the following use cases:
+
+* When connecting to {brandname} through a `LoadBalancer`, `NodePort`, or {openshiftshort} `Route`.
+* When an {brandname} Server cluster does not send internal and hidden cluster topology.
```

Comment:
```suggestion
* {brandname} Server does not send cluster topology to Hot Rod clients.
```

Reply:
Avoid putting an indefinite article before `{brandname}`. That attribute resolves to either Infinispan or Data Grid. So, downstream, you'd end up with "an Data Grid".